flink-table-store-core/src/main/java/org/apache/flink/table/store/file/operation/KeyValueFileStoreWrite.java

```diff
@@ -170,12 +173,7 @@ private MergeTreeWriter createMergeTreeWriter(
     }
 
     private boolean bufferSpillable() {
-        try {
-            return options.writeBufferSpillable(
-                    pathFactory.root().getFileSystem().getKind() != FileSystemKind.FILE_SYSTEM);
-        } catch (IOException e) {
-            throw new UncheckedIOException(e);
-        }
```

Comment:
Since `pathFactory` is only used at here, can we remove this field from the constructor of this class?